instrumentation/github.com/aws/aws-lambda-go/otellambda/lambda.go

```diff
@@ -24,6 +24,10 @@ import (
 
 	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/attribute"
+
+	// import to call ForceFlush method of batch_span_processor.go
+	//https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/trace/batch_span_processor.go#L167
+	_ "go.opentelemetry.io/otel/sdk/trace"
```

Comment:
```suggestion
```

This is unnecessary and must be removed.  Instrumentation packages must never depend on an SDK implementation.  The instrumentation defines its own `Flusher` interface so that it does not need to explicitly depend on any concrete type.

Reply:
If we remove this then we might need to add this method (https://github.com/open-telemetry/opentelemetry-go-contrib/pull/1413#discussion_r745082701) again right? I tested the code without it and we're not seeing instrumentation nodes coming up properly.

Reply:
No, there is no need for that.  This won't be released until after the next SDK release.  The `go.mod` files that need to import the SDK will be updated as appropriate at that time.  We do not make any guarantees regarding the ability to use unreleased commits of the SDK or instrumentation.